node/setup.go

```diff
@@ -229,7 +229,7 @@ func createPeerManager(
 		MaxConnectedUpgrade:    maxUpgradeConns,
 		MaxPeers:               maxUpgradeConns + 2*maxConns,
 		MinRetryTime:           250 * time.Millisecond,
-		MaxRetryTime:           30 * time.Minute,
+		MaxRetryTime:           1 * time.Minute,
```

Comment:
5 min max retry gives about 50 hours 
1 min gives about 2 hours before they're removed as a peer

Reply:
2 hours is still way too long. But the math seems to be wrong here?

I think it should be 
250ms + 500ms + 1s + 2s +4s + 8s + 16s + 32s + 60s ~= 124s

Reply:
I don't think it's doubling, rather its just adding 250ms each failure? 
```
	delay := m.options.MinRetryTime * time.Duration(failures)

```

Reply:
Actually I think your calculation is correct for the current logic, but it didn't match with the comment or what it intended to implement.

Let's fix the retry backoff to be real exponential

Reply:
Yeah, sounds good, btw where are you seeing the comment for exponential backoff?